scss/components/_case-custom-fields.scss

```diff
@@ -0,0 +1,88 @@
+.civicase__case-custom-fields__container {
+  float: left;
+  width: 50%;
+
+  &:first-child {
+    padding: 0 15px 24px 24px;
+  }
+
+  &:nth-child(2) {
+    padding: 0 24px 24px 15px;
+  }
+
+  .crm-editable-enabled:not(.crm-editable-editing) {
+    &:hover {
+      border: 2px dashed transparent;
+      padding: 15px 20px;
+    }
+  }
+
+  .crm-case-custom-form-block {
+    padding: 15px 20px;
+
+    table {
+      width: 100%;
+    }
+  }
+
+  .crm-submit-buttons {
+    border: 0;
+    padding: 15px 8px 0;
+    text-align: right;
+  }
+
+  .crm-form-submit {
+    min-width: auto;
+    padding: 7px 19px;
```

Comment:
can we use `padding: 7px 20px`. According to me units should be a multiple of 10.

Reply:
Default padding of bootstrap is added here. `crm-form-submit` is a core class, and we cannot really change the Markup to add btn class. That is why the same padding is applied to maintain consistency with other buttons.